src/main/resources/application.yaml

```diff
@@ -24,25 +24,32 @@ service:
     - name: bulkscan
       sas-validity: 300 #In seconds
       target-storage-account: bulkscan
+      target-container: bulkscan
     - name: cmc
       sas-validity: 300 #In seconds
       target-storage-account: bulkscan
+      target-container: cmc
     - name: crime
       sas-validity: 300 #In seconds
       enabled: ${CRIME_ENABLED:false}
       target-storage-account: crime
+      target-container: ${CRIME_DESTINATION_CONTAINER}
```

Comment:
won;t work if env var is not defined (and it's not in this branch)

Reply:
It is defined aleady